src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java

```diff
@@ -859,4 +859,10 @@ public void rhsOfDeclarationTransformed() throws Exception {
         assertRejected(new StaticWhitelist(), "staticMethod jenkins.model.Jenkins getInstance", "jenkins.model.Jenkins x = jenkins.model.Jenkins.getInstance()");
     }
 
+    @Issue("JENKINS-46191")
+    @Test
+    public void emptyDeclaration() throws Exception {
+        assertEvaluate(new GenericWhitelist(), "abc", "String a; a = 'abc'; return a");
+    }
```

Comment:
Those I believe we already have. But yeah, I need to spend some time locking in more thorough coverage.